politeiawww/www.go

```diff
@@ -293,6 +294,8 @@ func (p *politeiawww) handleLogout(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Revoke users authentication
+	session.Values["email"] = ""
+	session.Values["id"] = 0
```

Comment:
Maybe we should make constants of these session value strings (`email`, `id`, `admin`, etc), what do you think?

Reply:
I am ok with forcing the proper type.  We should only use them very rarely and it is ok to blow up if we do it wrong since it is sensitive data.